### PR TITLE
FLW-888 iOS: Update Subscan API URL, add token

### PR DESCRIPTION
**Pull request by stas-litvinov**
- change subscan URLs
- add API key

### CODE REVIEW

File: fearless/Common/Configs/ApplicationConfigs.swift [hunk 1]
```diff
@@ -107,4 +107,8 @@ extension ApplicationConfig: ApplicationConfigProtocol {
     var learnControllerAccountURL: URL {
         URL(string: "https://wiki.polkadot.network/docs/en/maintain-guides-how-to-nominate-polkadot#setting-up-stash-and-controller-keys")!
     }
+
+    var subscanAPIHeaderKey: String {
+        "X-API-Key"
```
Comment (ERussel) [inline fearless/Common/Configs/ApplicationConfigs.swift:112]: Just put this guy straight to the Request header. Motivation: this is not the application config but part of the protocol to communicate with subscan and should reside inside implementation. You could have a constant in the SubscanOperation factory if you wish :)

File: fearless/Modules/Staking/RecommendedValidators/RecommendedValidatorsViewController.xib [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="17701" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
```
Comment (ERussel) [inline fearless/Modules/Staking/RecommendedValidators/RecommendedValidatorsViewController.xib:2]: Are we changing also some styles in this PR?

Comment (stas-litvinov) [reply fearless/Modules/Staking/RecommendedValidators/RecommendedValidatorsViewController.xib]: Missed how it slipped into the PR. Rolled back

File: fearless/Common/Extension/Model/Chain+Info.swift [hunk 3]
```diff
@@ -48,11 +48,11 @@ extension Chain {
     func subscanExtrinsicURL(_ hash: String) -> URL? {
         switch self {
         case .polkadot:
-            return URL(string: "https://polkadot.subscan.io/extrinsic/\(hash)")
```
Comment (ERussel) [inline fearless/Common/Extension/Model/Chain+Info.swift:51]: This urls shouldn't be changed. They are just for opening the website

File: fearless/Common/Extension/Model/Chain+Info.swift [hunk 4]
```diff
@@ -61,11 +61,11 @@ extension Chain {
     func subscanAddressURL(_ address: String) -> URL? {
         switch self {
         case .polkadot:
-            return URL(string: "https://polkadot.subscan.io/account/\(address)")
```
Comment (ERussel) [inline fearless/Common/Extension/Model/Chain+Info.swift:64]: This urls shouldn't be changed. They are just for opening the website

**Review by ef1rspb — APPROVED** [review]

**Pull request merged**